sormas-operator/deploy/crds/sormas.hzi-braunschweig.com_v1alpha1_sormashelm_cr.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: sormas.hzi-braunschweig.com/v1alpha1
+kind: SormasHelm
+metadata:
+  name: example-sormashelm
```

Comment:
Clean up this bit

Reply:
Which part of it exactly?

Reply:
> Hi @nk-designz thanks for the contribution. Please fix the conflicts and Squash the commits into one. Looking forward to testing the operator.
> 
> Also include in the readme Description on how to deploy the operator on openshift and kubernetes.

* The conflict is my updated readme.
* The instructions are in the readme
* The commits are squashed like a mosquito

And the ```..._cr.yaml``` is a autogenerated sample by the operator-sdk for a test deployment of SORMAS

Have fun with it :D

Reply:
Now the conflict is resolved.
I wondered how to do that.

Reply:
Will merge and test it out tomorrow. Almost  12am here. Thanks for the good work.